app/scripts/controllers/preferences.js

```diff
@@ -376,6 +401,35 @@ class PreferencesController {
   //
   // PRIVATE METHODS
   //
+
+  /**
+   * Getter definition for the `tokens` property of store when controller is initialized
+   *
+   *
+   */
+  _defineTokens () {
+    const selectedAddress = this.store.getState().selectedAddress
+    const accountTokens = this.store.getState().accountTokens
+    const providerType = this.network.providerStore.getState().type
+    if (!(selectedAddress in accountTokens)) accountTokens[selectedAddress] = {}
```

Comment:
This line is in at least two other functions. We might want to DRY that up. Then again, if it's only one line, not much to reduce...

Reply:
good one, this is not necessary since the entry is created for each account when it is added, so the only place where it should be is on `addAddresses`.